src/libraries/System.IO.Pipelines/tests/PipeReaderCopyToAsyncTests.cs

```diff
@@ -286,5 +286,41 @@ public async Task ThrowingFromStreamCallsAdvanceToWithStartOfLastReadResult(int
             Assert.True(startPosition.Equals(wrappedPipeReader.LastConsumed));
             Assert.True(startPosition.Equals(wrappedPipeReader.LastExamined));
         }
+
+        [Fact]
+        public async Task CopyToAsyncStreamCopiesRemainderAfterReadingSome()
+        {
+            var buffer = Encoding.UTF8.GetBytes("Hello World");
```

Comment:
Normally I'd comment on changing `var` => `byte[]`, but it appears this file is already filled with such usage, so I won't say anything :wink:

Reply:
😆 habit